src/app/charts/core/templates/_helpers-sso.tpl

```diff
@@ -188,15 +190,10 @@ idp.xml
     {{- fail (printf "The OIDC configuration must be given as a map, not a %s: %s" (kindOf $oidc) $oidc) -}}
   {{- end -}}
 
-  {{- /* Set the usersDirectory for all clients to the default one ... */ -}}
-  {{- $baseUrl := (include "arkcase.tools.conf" (dict "ctx" $ctx "value" "baseUrl")) -}}
```

Comment:
See this ... referenced below

Reply:
Interesting ... with the code in the new location the lookup works.  The original code always returned  the default value from src/app/charts/core/values.yaml.

Reply:
That's odd indeed ... the `arkcase.tools.conf` template was meant to facilitate looking configuration values, specifically, no matter how/where they'd be housed. This makes it easy to fetch them without knowing their full whereabouts, while also giving the maintainers freedom to restructure the configuration if desired.

I'll have to look into why one worked but not the other ...